Samples/auth/Outlook-Add-in-SSO-events/manifest.xml

```diff
@@ -0,0 +1,149 @@
+                  <Control xsi:type="Button" id="AppointmentOrganizerSelectButton">
+                    <Label resid="SelectButtonLabel"/>
+                    <Supertip>
+                      <Title resid="SelectButtonLabel"/>
+                      <Description resid="SelectButtonDescription"/>
+                    </Supertip>
+                    <Icon>
+                      <bt:Image size="16" resid="Icon.16x16" />
+                      <bt:Image size="32" resid="Icon.32x32" />
+                      <bt:Image size="80" resid="Icon.80x80" />
+                    </Icon>
+                    <Action xsi:type="ShowTaskpane">
+                      <SourceLocation resid="Taskpane.Url"/>
+                    </Action>
+                  </Control>
+                </Group>
+              </OfficeTab>
+            </ExtensionPoint>
+            <!-- Enables the add-in on specified events. -->
+            <ExtensionPoint xsi:type="LaunchEvent">
+              <LaunchEvents>
+                <LaunchEvent Type="OnNewMessageCompose" FunctionName="onMessageComposeHandler"/>
+              </LaunchEvents>
+              <!-- Identifies the runtime to be used by the add-in (also referenced by the Runtime element). -->
+              <SourceLocation resid="WebViewRuntime.Url"/>
+            </ExtensionPoint>
+
+          </DesktopFormFactor>
+        </Host>
+      </Hosts>
+
+      <Resources>
+        <bt:Images>
+          <bt:Image id="Icon.16x16" DefaultValue="https://localhost:3000/assets/icon-16.png"/>
+          <bt:Image id="Icon.32x32" DefaultValue="https://localhost:3000/assets/icon-32.png"/>
+          <bt:Image id="Icon.80x80" DefaultValue="https://localhost:3000/assets/icon-80.png"/>
+        </bt:Images>
+        <bt:Urls>
+          <bt:Url id="JSRuntime.Url" DefaultValue="https://localhost:3000/commands/commands.js"/>
+          <bt:Url id="WebViewRuntime.Url" DefaultValue="https://localhost:3000/commands/commands.html"/>
+          <bt:Url id="Taskpane.Url" DefaultValue="https://localhost:3000/taskpane/taskpane.html"/>
+        </bt:Urls>
+        <bt:ShortStrings>
+          <bt:String id="CategoriesGroupLabel" DefaultValue="Outlook add-in SSO in events sample"/>
+          <bt:String id="SelectButtonLabel" DefaultValue="Show task pane"/>
+        </bt:ShortStrings>
+        <bt:LongStrings>
+          <bt:String id="SelectButtonDescription" DefaultValue="Opens the task pane for the Outlook add-in SSO in events sample."/>
+        </bt:LongStrings>
+      </Resources>
+      <WebApplicationInfo>
+        <Id>Enter_client_ID_here</Id>
+        <Resource>api://localhost:3000/Enter_client_ID_here</Resource>
+        <Scopes>
+          <Scope>openid</Scope>
+          <Scope>user.read</Scope>          
+          <Scope>profile</Scope>
+        </Scopes>
+      </WebApplicationInfo>
+      <ExtendedPermissions>
+        <ExtendedPermission>AppendOnSend</ExtendedPermission>
+      </ExtendedPermissions>
```

Comment:
Consider adding comments for \<WebApplicationInfo> and \<ExtendedPermissions> to match existing comments for event-based activation in the manifest.

Reply:
Added. thanks!